NitroxServer-Subnautica/Serialization/SubnauticaServerProtoBufSerializer.cs

```diff
@@ -2,8 +2,10 @@
 using NitroxModel.DataStructures.GameLogic.Buildings.Rotation;
 using NitroxModel_Subnautica.DataStructures.GameLogic;
 using NitroxModel_Subnautica.DataStructures.GameLogic.Buildings.Rotation;
+using NitroxModel_Subnautica.DataStructures.Surrogates;
 using NitroxServer.Serialization;
 using ProtoBufNet.Meta;
+using UnityEngine;
```

Comment:
To what extent do we still need this dependency in the sever? Just a question for future reference.

Reply:
Purely to help protobuf not crash on unknown types when it is reading the game's asset files. I don't know what the best solution is yet to remove the dependency on Unity/UWE code.